sql/nm_spawn_points.sql

```diff
@@ -5701,56 +5701,56 @@ INSERT INTO `nm_spawn_points` VALUES (17547490,0,7.549,2.599,118.240); -- Prowlo
-INSERT INTO `nm_spawn_points` VALUES (17555890,13,196.773,8.252,-308.707);
-INSERT INTO `nm_spawn_points` VALUES (17555890,14,212.375,8.006,-309.179);
-INSERT INTO `nm_spawn_points` VALUES (17555890,15,205.479,7.084,-284.712);
-INSERT INTO `nm_spawn_points` VALUES (17555890,16,226.669,7.500,-308.960);
-INSERT INTO `nm_spawn_points` VALUES (17555890,17,205.457,7.780,-299.290);
-INSERT INTO `nm_spawn_points` VALUES (17555890,18,229.418,7.924,-281.928);
-INSERT INTO `nm_spawn_points` VALUES (17555890,19,234.193,7.500,-327.967);
-INSERT INTO `nm_spawn_points` VALUES (17555890,20,229.321,8.725,-296.566);
-INSERT INTO `nm_spawn_points` VALUES (17555890,21,248.498,7.700,-287.928);
-INSERT INTO `nm_spawn_points` VALUES (17555890,22,212.373,7.891,-273.686);
-INSERT INTO `nm_spawn_points` VALUES (17555890,23,219.563,8.500,-280.989);
-INSERT INTO `nm_spawn_points` VALUES (17555890,24,235.472,6.751,-284.453);
-INSERT INTO `nm_spawn_points` VALUES (17555890,25,213.845,7.951,-273.953);
-INSERT INTO `nm_spawn_points` VALUES (17555890,26,226.365,8.500,-319.678);
-INSERT INTO `nm_spawn_points` VALUES (17555890,27,225.367,8.098,-296.425);
-INSERT INTO `nm_spawn_points` VALUES (17555890,28,200.939,7.846,-289.484);
-INSERT INTO `nm_spawn_points` VALUES (17555890,29,227.849,8.500,-320.373);
-INSERT INTO `nm_spawn_points` VALUES (17555890,30,196.482,8.226,-300.558);
-INSERT INTO `nm_spawn_points` VALUES (17555890,31,237.806,8.372,-304.311);
-INSERT INTO `nm_spawn_points` VALUES (17555890,32,236.669,7.483,-310.887);
-INSERT INTO `nm_spawn_points` VALUES (17555890,33,197.317,8.213,-294.733);
-INSERT INTO `nm_spawn_points` VALUES (17555890,34,194.723,7.272,-314.654);
-INSERT INTO `nm_spawn_points` VALUES (17555890,35,223.670,8.183,-278.056);
-INSERT INTO `nm_spawn_points` VALUES (17555890,36,215.315,8.183,-282.985);
-INSERT INTO `nm_spawn_points` VALUES (17555890,37,213.837,8.386,-277.434);
-INSERT INTO `nm_spawn_points` VALUES (17555890,38,239.121,7.477,-311.985);
-INSERT INTO `nm_spawn_points` VALUES (17555890,39,204.118,7.840,-296.797);
-INSERT INTO `nm_spawn_points` VALUES (17555890,40,204.809,7.510,-293.735);
-INSERT INTO `nm_spawn_points` VALUES (17555890,41,218.759,8.385,-314.973);
-INSERT INTO `nm_spawn_points` VALUES (17555890,42,218.131,8.442,-282.762);
-INSERT INTO `nm_spawn_points` VALUES (17555890,43,217.237,8.473,-315.697);
-INSERT INTO `nm_spawn_points` VALUES (17555890,44,219.195,8.338,-314.381);
-INSERT INTO `nm_spawn_points` VALUES (17555890,45,234.118,7.500,-319.658);
-INSERT INTO `nm_spawn_points` VALUES (17555890,46,243.304,7.563,-286.714);
-INSERT INTO `nm_spawn_points` VALUES (17555890,47,201.935,7.500,-323.092);
-INSERT INTO `nm_spawn_points` VALUES (17555890,48,209.553,7.118,-292.888);
-INSERT INTO `nm_spawn_points` VALUES (17555890,49,213.200,8.350,-316.937);
+INSERT INTO `nm_spawn_points` VALUES (17555763,0,228.000,7.134,-311.000);
+INSERT INTO `nm_spawn_points` VALUES (17555763,1,220.463,8.136,-302.294);
```

Comment:
later this month want to automate some labelling on these? maybe generate a new column to hold human readable names we don't actually use core side, just so it survives a future re-dump

Reply:
Can definitely make that happen (I like the idea of using the name string if possible instead of hard IDs)